src/styles/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.block {
+  width: $width;
+  margin: 0 auto;
+
+  .calendar {
+    margin-top: 50px;
```

Comment:
Calendar is not an element of 'block' BEM block
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/a4859588-4663-4bdf-ac1d-c3e95c760cb5)